grid2op/Environment/baseEnv.py

```diff
@@ -334,6 +334,7 @@ def __init__(
         highres_sim_counter=None,
         update_obs_after_reward=False,
         n_busbar=2,
```

Comment:
`n_busbar=2` should be `n_busbar = DEFAULT_N_BUSBAR_PER_SUB` (i know it's your mistake, it's mine, but i just spotted it :-)

Reply:
Fixed